### PR TITLE
SERVER-72296 Updated virtual workstation setup script to clone v6.0, …

### PR DESCRIPTION
**Pull request by IamXander**
…use newer python, and use correct boost pretty printers

### CODE REVIEW

**Comment by IamXander** [comment]
Tested this on a clean ubuntu workstation and works great!

TODO: update docs once this is merged

File: virtual_workstation_setup.sh [hunk 1]
```diff
@@ -170,32 +170,29 @@ setup_master() {
     echo "Finished setting up the mongo repo..."
 }
 
-setup_50() {
+setup_60() {
```
Comment (deriamis) [inline virtual_workstation_setup.sh:173]: Should we not *also* be setting up 5.0? I think the goal here was to have all current branches set up.

**Review by deriamis — APPROVED** [review]
One question, otherwise LGTM

File: virtual_workstation_setup.sh [hunk 2]
```diff
@@ -208,48 +205,7 @@ setup_50() {
         deactivate
         set -o nounset
     popd
-    echo "Finished setting up the 5.0 branch"
-}
-
-setup_44() {
```
Comment (Trevor159) [inline virtual_workstation_setup.sh:214]: was this unused before?

**Review by Trevor159 — APPROVED** [review]

File: virtual_workstation_setup.sh (hunk 2 above)
Comment (IamXander) [reply virtual_workstation_setup.sh:214]: Yes

File: virtual_workstation_setup.sh (hunk 1 above)
Comment (IamXander) [reply virtual_workstation_setup.sh:173]: Then we should setup v5, v4.4, and v4.2 but we didn't do that before. I am just keeping things consistent.

**Pull request merged**